stdlib/public/Platform/ucrt.swift

```diff
@@ -83,7 +82,3 @@ public let DBL_MIN = Double.leastNormalMagnitude
 
 @available(swift, deprecated: 3.0, message: "Please use 'Double.leastNonzeroMagnitude' or '.leastNonzeroMagnitude'.")
 public let DBL_TRUE_MIN = Double.leastNonzeroMagnitude
-
-public let M_LN2 = ucrt.M_LN2
-public let M_LOG10E = ucrt.M_LOG10E
-public let M_2_SQRTPI = ucrt.M_2_SQRTPI
```

Comment:
Q: Would it make sense to just hardcode these (i.e., `public let M_LN2 = 0.69314718055994529`), for compatibility instead of deleting them?

Reply:
This was not supposed to be removed, it needs to be restored.